docs/analyzer/customizing_nlp_models.md

```diff
@@ -63,9 +63,30 @@ Configuration can be done in two ways:
         -
         lang_code: es
         model_name: es_core_news_md 
+    ner_model_configuration:
+    labels_to_ignore:
+    - O
```

Comment:
whats the - 0? representing an empty list?

Reply:
can be `labels_to_ignore: []`

Reply:
It's the letter "O", which is "Out of entity" in NER.